### PR TITLE
Bump k8s to 1.16.2

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
ci test please

**Comment by qinqon** [comment]
Maybe newer nm version at nodes

```
NM main-loop aborted: Connection update failed: error=nm-connection-error-quark: ipv4.gateway: gateway cannot be set if there are no addresses configured (7), dev=eth0/<enum NM_DEVICE_STATE_ACTIVATED of type NM.DeviceState>

```

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
Darn. for me it works locally too.

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
 
 unit_test_args ?=  -r --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_ARGS)
 
-export KUBEVIRT_PROVIDER ?= k8s-1.15.1
+export KUBEVIRT_PROVIDER ?= k8s-1.16.2
```
Comment (phoracek) [inline Makefile]: Please bump the version also in https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/developer-guide-commands.md#local-cluster. It is there on two places.

**Pull request closed without merging**